app/services/course_search_service.rb

```diff
@@ -52,6 +52,9 @@ def sites_with_distance_from_origin
     # only want new and running sites
     new_and_running_sites = sites_with_status.where(site_status[:status].in(%w[new_status running]))
 
+    # we only want sites that have been geocoded
+    new_and_running_sites = new_and_running_sites.where(sites[:latitude].not_eq(nil).and(sites[:longitude].not_eq(nil)))
```

Comment:
The meaning of this variable name doesn't seem right with the additional `where`.

Reply:
I have now renamed some local variables